pkg/daemon/discover/discover.go

```diff
@@ -198,9 +199,9 @@ func udevBlockMonitor(c chan string, period time.Duration) {
 	// get discoverDaemonUdevBlacklist from the environment variable
```

Comment:
Update comment to match the logic in code

Reply:
Will do